tests/platform_tests/api/test_sfp.py

```diff
@@ -161,6 +184,10 @@ def is_xcvr_optical(self, xcvr_info_dict):
                 compliance_code = spec_compliance_dict.get("10/40G Ethernet Compliance Code")
                 if compliance_code == "40GBASE-CR4":
                    return False
+                if compliance_code == "Extended":
+                    extended_code = spec_compliance_dict.get("Extended Specification Compliance")
+                    if "CR" in extended_code:
```

Comment:
There other extended code as well [https://github.com/Azure/sonic-platform-common/blob/master/sonic_platform_base/sonic_xcvr/api/public/sff8636.py#L303](https://github.com/Azure/sonic-platform-common/blob/master/sonic_platform_base/sonic_xcvr/api/public/sff8636.py#L303) . Could you add them as well?